# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/systemd/hotfix-no-fuzz.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- systemd-256/test/fuzz/meson.build.vanilla	2024-06-27 19:37:37.722000000 +0200
+++ systemd-256/test/fuzz/meson.build	2024-06-27 19:36:32.859000000 +0200
@@ -90,6 +90,6 @@
         fuzz_in = fs.name(p)
 
         dict = fuzz_regression_tests.get(fuzzer, {})
-        dict += { 'files' : dict.get('files', []) + files(fuzzer / fuzz_in) }
+        #dict += { 'files' : dict.get('files', []) + files(fuzzer / fuzz_in) }
         fuzz_regression_tests += { fuzzer : dict }
 endforeach
